src/components/tables/RefreshTokens/Rows.tsx

```diff
@@ -0,0 +1,47 @@
+import { TableCell, TableRow, useTheme } from '@mui/material';
+import { RefreshTokenQuery } from 'api/tokens';
+import { getEntityTableRowSx } from 'context/Theme';
+import { FormattedMessage } from 'react-intl';
+import TimeStamp from '../cells/TimeStamp';
+import RevokeTokenButton from '../cells/refreshTokens/RevokeToken';
+
+interface RowsProps {
+    data: RefreshTokenQuery[];
+}
+
+interface RowProps {
+    row: RefreshTokenQuery;
+}
+
+function Row({ row }: RowProps) {
+    const theme = useTheme();
+
+    return (
+        <TableRow hover sx={getEntityTableRowSx(theme)}>
+            <TimeStamp time={row.created_at} enableRelative />
+
+            <TableCell>{row.detail}</TableCell>
```

Comment:
Please put a max width and wrap text on this column to prevent the _very very very rare_ chance that the string is super long.

![image](https://github.com/estuary/ui/assets/270078/79fd5b98-7887-4d16-9e8e-c64f63c88b25)